util/dvsim/SlurmLauncher.py

```diff
@@ -46,28 +47,30 @@ def _do_launch(self):
 
         self._dump_env_vars(exports)
 
+        # Add a command delimiter if necessary
+        slurm_setup_cmd = SLURM_SETUP_CMD
+        if slurm_setup_cmd != '' and not slurm_setup_cmd.endswith(';'):
```

Comment:
Nit: can just be "if slurm_setup_cmd and ...", since an empty string evaluates to False in a conditional.